ee/vellum_ee/workflows/display/utils/expressions.py

```diff
@@ -245,6 +246,15 @@ def serialize_key(key: Any) -> str:
 
 
 def serialize_value(display_context: "WorkflowDisplayContext", value: Any) -> JsonObject:
+    if value is undefined:
```

Comment:
The difference between `undefined` and `null` is that `undefined` should always be skipped from the JSON wherever it's resolved. For example:
```
{"foo": "bar", "baz": null}
```

vs

```
{"foo": "bar"}
```